src/main/java/analyzer/exercises/Twofer.java

```diff
@@ -36,14 +40,31 @@ public void parse() {
             } else if (walker.usesFormat) {
                 this.statusObject.put("status", "disapprove_with_comment");
                 this.comments.put("java.two-fer.stringFormatPerformance");
-            } else if (walker.returnCount > 1) {
+            } else if (walker.returnStmts.size() > 1) {
                 this.statusObject.put("status", "disapprove_with_comment");
                 this.comments.put("java.two-fer.multipleReturns");
             } else {
                 if (walker.usesIfStatement) {
                     this.comments.put("java.two-fer.useTernaryExpression");
                 }
-
+                if (walker.returnStmts.size() == 1) {
+                    if (OptimalFormat.checkReturn(walker.returnStmts)) {
```

Comment:
These should implement a common interface of some type so that we can iterate through them quickly no matter how many there are. Perhaps it should be an enum where each value implements a `checkReturn` method? The `checkReturn` methods can delegate to utility classes like OptimalFormat, etc.

In any case, there is too much duplication here of the same lines 4 times.